### DISCUSSION TITLE
Official citation?

### DISCUSSION THREAD

**Discussion by choi328328** [NVIDIA/NVFlare#695] [Q&A] [answered]
Hi. My name is Byungjin Choi, and I am a medical informatician in South Korea.

I am preparing to publish a machine learning study using nvflare in the medical journal.
However, I couldn't find an official guideline for citing nvflare.

How should I cite nvflare? 
Or is there a paper or conference presentation related to nvflare that I can cite instead?

**Comment by YuanTingHsieh** [comment]
Hi Byungjin (@choi328328 )

We are working on the official citation link right now.
It should come out within this week.
If you need to submit the paper before the link comes out, maybe you can just cite the "https://github.com/NVIDIA/NVFlare" link. (and change it later when paper is accepted.)

Thanks for bringing this issue!

**Comment by YuanTingHsieh** [comment, marked as answer]
Hi Byungjin (@choi328328 )

We've added the citation file.
You can cite this repo using "Cite this repository" on the right-hand side of https://github.com/NVIDIA/nvflare 

You can refer to here for details:
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files

**Comment by holgerroth** [comment]
Note, the [citation file](https://github.com/NVIDIA/NVFlare/blob/dev/CITATION.cff) has recently been updated to cite our framework paper
```
@article{Roth_NVIDIA_FLARE_Federated_2022,
author = {Roth, Holger R. and Cheng, Yan and Wen, Yuhong and Yang, Isaac and Xu, Ziyue and Hsieh, Yuan-Ting and Kersten, Kristopher and Harouni, Ahmed and Zhao, Can and Lu, Kevin and Zhang, Zhihong and Li, Wenqi and Myronenko, Andriy and Yang, Dong and Yang, Sean and Rieke, Nicola and Quraini, Abood and Chen, Chester and Xu, Daguang and Ma, Nic and Dogra, Prerna and Flores, Mona and Feng, Andrew},
doi = {https://doi.org/10.48550/arXiv.2210.13291},
journal = {International Workshop on Federated Learning, NeurIPS 2022, New Orleans, USA},
month = {10},
title = {{NVIDIA FLARE: Federated Learning from Simulation to Real-World}},
year = {2022}
}
```
